pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -105,8 +105,18 @@ func makeSimpleImportSpans(
 		}
 
 		spanCoverStart := len(cover)
-
 		for layer := range backups {
+			isCovered, err := data.coveredByLaterReintroduction(span.Key, backups[layer].EndTime)
```

Comment:
nit: I'd call this coveredLater; if anything it is the opposite of covered right now, right?